pcmanfm/tabpage.h

```diff
@@ -280,6 +280,10 @@ class TabPage : public QWidget {
 
     void ceateShortcut();
 
+    void setFilesToSelect(QStringList files) {
+        filesToSelect_ = files;
+    }
```

Comment:
Please change `QStringList` to `const QStringList&`  and also sort the list, like:

```c++
    void setFilesToSelect(const QStringList& files) {
        filesToSelect_ = files;
        filesToSelect_.removeDuplicates();
    }
```

Reply:
Sorry, I meant removing duplicate items but wrote "sorting". Corrected now.

Reply:
Got it, already pushed, sorry if I make a lot commits/push too fast haha